app/models/rspec-test/spec/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../../user'
+
+RSpec.describe User do
+  describe '#id' do
```

Comment:
Idiomatic

describeにはテストの対象となるメソッド名を書くようにしよう！
（テスト実行時に出てくる標準出力が、テストの仕様書として機能するために必要なので。）

```suggestion
  describe '#present?' do
```